extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/ticket/TicketValidationService.java

```diff
@@ -75,6 +84,10 @@ public String processUsername(String ticket) throws GuacamoleException {
         catch (TicketValidationException e) {
             throw new GuacamoleException("Ticket validation failed.", e);
         }
+        catch (Throwable t) {
+            logger.error("Error validating ticket with CAS server: {}", t.getMessage());
+            throw new GuacamoleInsufficientCredentialsException("Error validating ticket with CAS server.", t, CredentialsInfo.USERNAME_PASSWORD);
```

Comment:
Internal errors really shouldn't be exposed to users of the system. If the ticket can't be validated, then that should definitely be logged for the administrator's sake, but such internals shouldn't make their way to the interface.

Why not let the interface continue rendering all failures as "Invalid login"?